remme/atomic_swap_tp/rest_api.py

```diff
@@ -5,47 +5,51 @@
     get_swap_expire_payload, get_swap_set_secret_lock_payload, get_swap_close_payload
 from google.protobuf.json_format import MessageToJson
 
-client = AtomicSwapClient()
-
 LOGGER = logging.getLogger(__name__)
 
 
 def init(**data):
     data = data['payload']
     payload = get_swap_init_payload(**data)
+    client = AtomicSwapClient()
```

Comment:
Why do we need this here and hereafter?

Reply:
It had some problem with a node key when I tested on py.test. Will undo this.